src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6070,6 +6070,14 @@ void handleMemberInitializer(IMemberInitializerOperation memberInitializer)
                 // We therefore visit the InitializedMember to get the implicit receiver for the contained initializer, and that implicit receiver will be cloned everywhere it encounters
                 // an IInstanceReferenceOperation with ReferenceKind InstanceReferenceKind.ImplicitReceiver
 
+                if (onlyContainsEmptyLeafNestedInitializers(memberInitializer))
+                {
+                    // However, when the leaf nested initializers are empty, we won't access the chain of initialized members
+                    // and we only evaluate the arguments/indexes they contain.
+                    addIndexes(memberInitializer);
```

Comment:
>addIndexes(memberInitializer); [](http://example.com/codeflow?start=20&length=30)

It would be good to have a test for a scenario when we get here in between of two meaningful initializers.